pkg/sql/table.go

```diff
@@ -114,6 +114,10 @@ func (p *planner) createOrUpdateSchemaChangeJob(
 		)
 	}
 
+	// p.extendedEvalCtx.SchemaChangeJobRecords refers to schemaChangeJobRecords
+	// `of extraTxnState` in `sql.connExecutor`. `connExecutor.createJobs` is the
```

Comment:
the backticks here are a little weird to me. Can you take another pass on this comment?